harambe/core.py

```diff
@@ -268,6 +269,18 @@ async def capture_pdf(
         )
         return res[0]
 
+    async def save_cookies(self) -> None:
+        """
+        Save the cookies from the current browser context.
+
+        This function retrieves all the cookies from the current browser context and
+        saves them to the SDK instance. It also notifies all the observers about the
+        action performed.
+        """
+        cookies = await self.page.context.cookies()
```

Comment:
lets make the cookies an optional param. If it is not provided, we will just save the page cookies